internal/controller/config/config.go

```diff
@@ -358,3 +361,18 @@ func (config *ProgressiveConfig) GetChildStatusAssessmentSchedule(kind string) (
 	}
 	return AssessmentSchedule{}, fmt.Errorf("No Assessment Schedule found for kind %q", kind)
 }
+
+// GetAnalysisRunTimeout parses the GlobalConfig for the analysisRunTimeout string to convert into a time.Duration
+func (config *ProgressiveConfig) GetAnalysisRunTimeout() (time.Duration, error) {
+
+	// default timeout is 20 minutes
+	defaultAnalysisRunTimeout := 1200
+
+	analysisRunTimeout, err := strconv.Atoi(config.AnalysisRunTimeout)
+	if err != nil {
+		return time.Duration(defaultAnalysisRunTimeout), fmt.Errorf("invalid analysisRunTimeout value: %w", err)
+	}
```

Comment:
won't this return an error if it's empty string? we don't want that. just return the default if it is